### ISSUE TITLE
NVDA does not announce "Read-Only" status in Notepad++ 8.8.5

### ISSUE THREAD

**Issue by hdzrvcc0X74** [notepad-plus-plus/notepad-plus-plus#17036]
### Is there an existing issue for this?

- [x] I have searched the existing issues

## Description

When opening a document in **Notepad++** that is set to **read-only**, NVDA does not announce the read-only status. This prevents users with visual impairments from knowing whether the document can be edited. I am not sure if there is any visual confirmation for this, but a blind user cannot know. 

## Additional information
- Notepad++ version: 8.8.5  
- NVDA version: 2025.3  
- OS: Windows 11 24H2 (AMD64, build 26100.6584)  

## Possible solutions (optional)
- Add a **UIA (UI Automation) property** that indicates read-only status, so NVDA can detect it.  
- Optionally, append “(read-only)” to the document title in Notepad++.

### Steps To Reproduce

1. Open Notepad++ 8.8.5.  
2. Open a document and set it to **read-only** via either:  
   - Notepad++: `Edit` → `Read-Only in Notepad++`;  
   - Notepad++: `Edit` → `Read-Only Attribute in Windows` (if available), or  
   - Windows: context menu > `Properties` (`Alt+Enter`) → `Read-only`.  
3. Run NVDA 2025.3.  
4. Try to determine the read-only status of the document using the screen reader.  

### Current Behavior

NVDA does not announce the read-only status, and the user cannot differentiate between read-only and editable documents.

### Expected Behavior

NVDA should clearly announce that the document is **read-only**, so blind users can know its status.

### Debug Information

```shell
Notepad++ v8.8.5   (64-bit)
Build time: Aug 14 2025 - 00:32:39
Scintilla/Lexilla included: 5.5.7/5.4.5
Boost Regex included: 1_85
Path: C:\Program Files\Notepad++\notepad++.exe
Command Line: C:\Users\Tarik\Documents\Posao\Programiranje\Kalkulator\potencija.py
Admin mode: OFF
Local Conf mode: OFF
Cloud Config: OFF
Periodic Backup: ON
Placeholders: ON
Scintilla Rendering Mode: SC_TECHNOLOGY_DIRECTWRITE (1)
Multi-instance Mode: monoInst
asNotepad: OFF
File Status Auto-Detection: cdEnabledNew (for current file/tab only) + cdGo2end
Dark Mode: ON
Display Info:
    primary monitor: 1920x1080, scaling 100%
    visible monitors count: 1
    installed Display Class adapters: 
        0000: Description - AMD Radeon(TM) Graphics
        0000: DriverVersion - 31.0.12027.9001
OS Name: Windows 11 Pro (64-bit)
OS Version: 24H2
OS Build: 26100.6584
Current ANSI codepage: 1252
Plugins: none
```

**Comment by xomx** [comment]
> * Add a **UIA (UI Automation) property** that indicates read-only status, so NVDA can detect it.

Related:
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/15759#issuecomment-2461845142
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/6060#issuecomment-1962524911
